draft-terbu-sd-jwt-vc.md

```diff
@@ -255,6 +255,12 @@ exist between `sub` and `cnf` claims.
 Additional public claims MAY be used in SD-JWT VCs depending on the
 application.
 
+#### SD-JWT VC without selectively disclosable claims
+
+SD-JWT VC may not contain any selectively disclosable claims.
+Such SD-JWT VC does not contain `_sd` claim in the JWT body and
+does not have any Disclosures.
```

Comment:
out of curiosity - the change to align issuance and presentation format would lead to the fact that non sd VCs would be JWTs + a `~`, right?

Reply:
I guess so, also key binding would be very easy as it is still being appended after ~ in presentation?

Reply:
aaa that is an important question. It is a little counter intuitive, but if sd-jwt is the basis of jwt vcs ending with Tilde will be more natural and will be a nice differentiator between normal jwts and jwt vcs..?